samples/nfc/record_launch_app/sample.yaml

```diff
@@ -9,6 +9,13 @@ tests:
       - nrf52dk_nrf52832
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuapp_ns
-    platform_allow: nrf52840dk_nrf52840 nrf52dk_nrf52832 nrf5340dk_nrf5340_cpuapp
+      - nrf54h20dk_nrf54h20_cpuapp
+      - nrf54l15pdk_nrf54l15_cpuapp
+    platform_allow: >
```

Comment:
? why `>` and not simply use te same scheme as above?

Reply:
This is done on purpose to make it more readable since we add more and more targets and single line won't be readable in my opinion. This yaml syntax makes all entry under platform_allow: > as they would be a single line for parser